src/utils/file_management.rs

```diff
@@ -0,0 +1,36 @@
+use std::collections::HashMap;
+
+use anyhow::{bail, Result};
+use colored::Colorize;
+use walkdir::WalkDir;
+
+pub fn get_files_in(dir: &str) -> Result<HashMap<String, Vec<u8>>> {
```

Comment:
@BoisterousCoder minor: but is this needed still? I think the ipfs client add, adds dirs?

Reply:
![Screenshot from 2023-01-19 15-16-51](https://user-images.githubusercontent.com/4443085/213562370-6bad9267-45ad-48a8-a54f-d427fc838d05.png)